orchestrator/domain/base.py

```diff
@@ -891,6 +904,10 @@ def db_model(self) -> SubscriptionInstanceTable:
     def parents(self) -> List[SubscriptionInstanceTable]:
         return self._db_model.parents
 
+    @property
+    def parent_ids(self) -> List[SubscriptionInstanceTable]:
+        return [c.subscription_id for c in self.parents]
```

Comment:
Should this function not already take care of the `parent_id != self.owner_subscription_id` check, so that `model.dict()["parent_ids"]` returns the same as `model.parent_ids`?

Reply:
This change led to DB Session issues hence the removal of this @prroperty